jenkins/janitor.py

```diff
@@ -0,0 +1,151 @@
+#!/usr/bin/env python
+
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import collections
+import datetime
+import json
+import pytz
```

Comment:
I might read it wrong - [http://pythonhosted.org/pytz/](http://pythonhosted.org/pytz/) so I indeed need to run `easy_install --upgrade pytz` ? Seems my mac has the library already.